common/src/View/MapFrame.h

```diff
@@ -76,7 +76,7 @@ namespace TrenchBroom {
             MapFrame();
             MapFrame(FrameManager* frameManager, MapDocumentSPtr document);
             void Create(FrameManager* frameManager, MapDocumentSPtr document);
-            ~MapFrame();
+            virtual ~MapFrame();
```

Comment:
The missing virtual keyword is only necessary if MapFrame were to be subclassed, which AFAIK it isn't. It certainly can't hurt though.